ocf/ocf.go

```diff
@@ -379,7 +379,7 @@ func newEncoder(schema avro.Schema, w io.Writer, cfg encoderConfig) (*Encoder, e
 				return nil, err
 			}
 
-			writer := avro.NewWriter(w, 512, avro.WithWriterConfig(cfg.EncodingConfig))
+			writer := avro.NewWriter(w, 512, avro.WithWriterConfig(avro.DefaultConfig))
```

Comment:
This does not fix anything. It is simple enough for the DefaultConfig to be overridden to change the tag.

Reply:
This is clearly a bug, easily verified with the provided code above., it's using override config 
ocf.NewEncoder(schemaStr, &buf, ocf.WithEncodingConfig(config))